subsys/bluetooth/host/l2cap.c

```diff
@@ -494,7 +513,7 @@ void bt_l2cap_send_cb(struct bt_conn *conn, u16_t cid, struct net_buf *buf,
 	hdr->len = sys_cpu_to_le16(buf->len - sizeof(*hdr));
 	hdr->cid = sys_cpu_to_le16(cid);
 
-	bt_conn_send_cb(conn, buf, cb, user_data);
+	return bt_conn_send_cb(conn, buf, cb, user_data);
```

Comment:
We can't ignore this since it does return errors in case syswq is used.

Reply:
I mean wherever this function is used (`bt_l2cap_send_cb`). The return value is currently being ignored 15 places in att.c for example.

Reply:
att.c uses a fixed channel though, so it can be ignored due to the fact we just block when using an RX thread or you are suggesting to start changing all of that now?